orca-core/src/main/groovy/com/netflix/spinnaker/orca/config/OrcaConfiguration.java

```diff
@@ -155,4 +156,9 @@ public StageDefinitionBuilderFactory stageDefinitionBuilderFactory(Collection<St
     return new DefaultStageDefinitionBuilderFactory(stageDefinitionBuilders);
   }
 
+  @Bean
+  public RetrySupport retrySupport() {
+    return new RetrySupport();
```

Comment:
Thinking that someday we might actually set some default retry/backoff values and avoid the need to choose randomly when being used.